README.md

```diff
@@ -1,2 +1,44 @@
 ## Astoria Digital's Design Directory
 - [Style Guide](https://www.figma.com/file/LXKWblcPXToV1uVakYXbhB/Astoria-Digital?node-id=133%3A3) by [@lifualexfan](http://github.com/lifualexfan)
+
+## Typefaces
+
+## Primary Typeface
+
+[DM Mono](https://fonts.google.com/specimen/DM+Mono)
+
+## Headlines
+
+### H1, H2
+[Fjalla One](https://fonts.google.com/specimen/Fjalla+One) 
+
+```
+
```

Comment:
```suggestion
```

Reply:
Thank you Chris! I've applied the changes.